pkg/controller/secret/secret_controller.go

```diff
@@ -150,6 +150,8 @@ func createPagerdutyRoute() *alertmanager.Route {
 
 		// https://issues.redhat.com/browse/OSD-1922
 		{Receiver: receiverMakeItWarning, Match: map[string]string{"alertname": "KubeAPILatencyHigh", "severity": "critical"}},
+		// Lower the alerts till we upgrade to version later than 4.5.17 https://bugzilla.redhat.com/show_bug.cgi?id=1885688
+		{Receiver: receiverMakeItWarning, Match: map[string]string{"alertname": "HAProxyReloadFail", "severity": "critical"}},
```

Comment:
@bmeng could you please update this to receiverNull ? we can then commit this with custom SRE alert.

Reply:
Updated.